src/io/flutter/logging/FlutterLogTree.java

```diff
@@ -207,6 +210,7 @@ private TreeModel(@NotNull ColumnModel columns, @NotNull Disposable parent, @Not
       setShowSequenceNumbers(false);
 
       uiThreadAlarm = new Alarm(Alarm.ThreadToUse.SWING_THREAD, parent);
+      updateTimer = new Timer(100, e -> uiExec(this::update, 0));
```

Comment:
consider using AsyncRateLimiter instead of manually creating the timer. It is designed for the problem of ensuring you don't overdo UI updates

Reply:
ah!  Crazy I overlooked that.  I'm not super familiar with the nuances here... Can you explain what tangible difference it makes over a Swing Timer?

Thanks!

Reply:
Nevermind.  I get it now.  Will migrate.

🍻